samples/snippets/cpp/VS_Snippets_Wpf/System.Windows.Interop.D3DImage/cpp/d3dhost.csproj

```diff
@@ -14,6 +14,7 @@
     <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <WarningLevel>4</WarningLevel>
     <TargetFrameworkProfile>Client</TargetFrameworkProfile>
+    <EnableNetAnalyzers>true</EnableNetAnalyzers>
```

Comment:
This might not work for legacy csproj files. See https://github.com/dotnet/roslyn-analyzers/issues/4776